src/components/ebay-combobox-readonly/index.marko

```diff
@@ -0,0 +1,53 @@
+$ var data = component.getTemplateData(state, input);
```

Comment:
I think we should do as before. 
Use input instead of data. 
We prob can move getTemplateData to template itself
For all the classes in template just do
`class=['class1', input.borderless && 'borderlessClass']` etc

Reply:
Updated. Removed `getTemplateData()` altogether.